### ISSUE TITLE
save() method with laravel5 new filesystem methodology 

### ISSUE THREAD

**Issue by tzookb** [Intervention/image#374]
I used the intervention package before for storing my images in to the local server.

Now I want to use the new laravel 5 filesystem methodology,
so I need to use their method:

     $filesystem->put($fileName, $contents);

but what is the contents of the generated intervention image?
I create the intervention image like this:

     $image->save($fileName);

but when I do that, it suppose to create the file by itself on the specific path the filename holds,
and I dont want to store it with intervention, I only want to generate the image and store it with filesystem.

So bottom line,
is there a way to get the contents of the generated image?

**Comment by tzookb** [comment]
Found it after 2 min :/

it is simply 

     $contents = $image->encode();

thanks!
